### DISCUSSION TITLE
BusinessFunction empty ?

### DISCUSSION THREAD

**Discussion by WickWine** [RehanSaeed/Schema.NET#60] [Q&A]
No members.. 
https://schema.org/BusinessFunction

**Comment by RehanSaeed** [comment]
I think this is because the values are from `purl.org`. If you want to add support for that, I'd love to know what is involved and maybe accept a PR for this work.

**Comment by WickWine** [comment]
I presume it would be similar to OfferItemCondition.cs except buy, sell etc resolving to the appropriate URL's ie http://purl.org/goodrelations/v1#Sell
As per http://www.heppnetz.de/ontologies/goodrelations/v1

**Comment by RehanSaeed** [comment]
Correct!

Closing for now. Feel free to reopen if you have anything further you want to add.
